src/olympia/reviewers/serializers.py

```diff
@@ -62,11 +62,10 @@ class FileEntriesSerializer(FileSerializer):
     filename = serializers.SerializerMethodField()
 
     class Meta:
-        fields = FileSerializer.Meta.fields + (
```

Comment:
so, maybe `FileEntriesSerializer` shouldn't inherit from `FileSerializer` any longer if it's not using any of it's fields or methods?

Reply:
Hmm, good point. Let me look into that.

Reply:
Fixed!